app/src/main/java/com/example/kitchenflow/ui/main/MainFragment.kt

```diff
@@ -68,6 +82,24 @@ class MainFragment : Fragment() {
             }
             adapter = OrdersAdapter(requireContext())
             ordersListRv.adapter = adapter
+            searchBarEt.addTextChangedListener(object : TextWatcher {
```

Comment:
Before I suggested moving the entire TextWatcher to ViewModel. You can do it by custom databinding property